src/include/distributed/connection_management.h

```diff
@@ -112,24 +97,25 @@ extern HTAB *ConnectionHash;
 extern struct MemoryContextData *ConnectionContext;
 
 
-extern void AfterXactConnectionHandling(bool isCommit);
+extern void AfterXactResetConnections(bool isCommit);
 extern void InitializeConnectionManagement(void);
 
 
 /* Low-level connection establishment APIs */
-extern MultiConnection * GetNodeConnection(uint32 flags, const char *hostname,
-										   int32 port);
-extern MultiConnection * StartNodeConnection(uint32 flags, const char *hostname,
-											 int32 port);
-extern MultiConnection * GetNodeUserDatabaseConnection(uint32 flags, const char *hostname,
-													   int32 port, const char *user, const
-													   char *database);
-extern MultiConnection * StartNodeUserDatabaseConnection(uint32 flags,
-														 const char *hostname,
+extern MultiConnection * GetNodeConnection(const char *hostname, int32 port,
+										   uint32 flags);
+extern MultiConnection * StartNodeConnection(const char *hostname, int32 port,
+											 uint32 flags);
+extern MultiConnection * GetNodeUserDatabaseConnection(const char *hostname, int32 port,
```

Comment:
Is there a reason you put flags last? Because it's the common parameter between all these - and will be common for the placement stuff too - it seems better to have it first?